client/catapult/CMakeLists.txt

```diff
@@ -114,3 +114,8 @@ if(EXISTS "${CMAKE_SOURCE_DIR}/internal/CMakeLists.txt")
 endif()
 
 add_subdirectory(tools)
+
+if(USE_CONAN)
+	message("--- copy conan deps ---")
+	install(DIRECTORY ${CMAKE_BINARY_DIR}/deps/ DESTINATION deps)
+endif()
```

Comment:
why is this needed? isn't this logic in CatapultConan.generate ?

Reply:
``CatapultConan.generate`` copies all the dependencies libs from their Conan package to the catapult build folder.
When ``cmake install`` or ``ninja install`` is run, the above code will copy the deps folder from the catapult build folder to the install folder.